examples/todo-list/src/__tests__/unit/controllers/todo-list-todo.controller.unit.ts

```diff
@@ -98,7 +101,7 @@ describe('TodoController', () => {
 
   describe('patch()', () => {
     it('returns a number of todos updated', async () => {
-      patch.resolves([aChangedTodo].length);
+      patch.resolves({count: [aChangedTodo].length});
```

Comment:
Nice! IIUC, the stricter type checks are detecting programming errors we were not aware of 👍